ci/testsettings-psql.py

```diff
@@ -0,0 +1,50 @@
+# This file is exec'd from settings.py, so it has access to and can
```

Comment:
is this one needed? I don't see it referenced in the build

Reply:
woops no this is left over from an earlier solution